021-IdentityForApps/Coach/09-custom-policy.md

```diff
@@ -0,0 +1,20 @@
+# Challenge \#9 - Creating Custom Policies
+
+[< Previous Challenge](./08-prepare-ief.md) - **[Home](./README.md)** - [Next Challenge>](./10-appinsights.md)
+## Introduction
+
+In this challenge, we'll deploy a custom policy to allow a user to Delete Their Account.
+
+This challenge uses a custom policy found in the Azure AD B2C Custom Policy gallery GitHub Repo, linked in the challenge.
```

Comment:
Do you want to mention here that you have files in the Solutions/Challenge-09 folder for them to look at?

Reply:
Good call - mention made